package.json

```diff
@@ -45,7 +45,9 @@
     "test:visual:native": "npm run build && playwright test --grep @visual-regression",
     "test:visual:refresh": "scripts/run-playwright-container.sh bash -c 'npm install && npm run test:visual:native -- --update-snapshots'",
     "test:watch": "vitest",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "release:alpha": "release-it --preRelease=alpha",
+    "release:stable": "release-it --git.tagMatch='v[0-9]*\\\\.[0-9]*\\\\.[0-9]*'"
```

Comment:
`--git.tagMatch` should make sure a stable release gets a full changelog (note that it's a glob rather than a regex).